index.html

```diff
@@ -17,13 +17,20 @@
     <section class="main-section">
       <header>
         <nav>
-          <a href="./index.html">Ashna Ali</a>
-          <ul class="desktop-ul">
-            <li class="desktop-link"><a href="#">Portfolio</a></li>
-            <li class="desktop-link"><a href="#">About</a></li>
-            <li class="desktop-link"><a href="#">Contact</a></li>
-          </ul>
-          <img src="./Union.jpg" alt="menu" class="menu-icon">
+          <a href="./index.html" id="logo">Ashna</a>
+          <div class="desktop-ul">
+            <div class="desktop-link"><a href="#Portfolio">Portfolio</a></div>
+            <div class="desktop-link"><a href="#About">About</a></div>
+            <div class="desktop-link"><a href="#Contact">Contact</a></div>
+          </div>
+          <div class="menu-icon">
+            <div class="item-closed"><a href="#" class="closing"><i class="fa fa-times" aria-hidden="true"></i></a></div>
+            <div class="mobile-nav"><a href="#Portfolio" class="closing">Portfolio</a></div>
+            <div class="mobile-nav"><a href="#About" class="closing">About</a></div>
+            <div class="mobile-nav"><a href="#Contact" class="closing">Contact</a></div>
```

Comment:
- [ ] Unfortunately, your menu links are not redirecting the user to the correct part of the page. Kindly make sure the `href` attribute value is correct and that the sections on the page have the same `id` values.

Reply:
@leonardodiasb  Can you help me I don't know why it's not working I tried a lot but all in vain.

Reply:
Hi, I just cloned your repo, and when I click on the links they redirect to the correct section, do you still have any thing you are not clear about?